src/main/scala/com/databricks/labs/overwatch/pipeline/Schema.scala

```diff
@@ -725,6 +734,31 @@ object Schema extends SparkSessionWrapper {
     StructField("workspace_name", StringType, nullable = true)
   ))
 
+  val warehouseSnapMinimumSchema: StructType = StructType(Seq(
+    StructField("warehouse_id", StringType, nullable = false),
+    StructField("name", StringType, nullable = true),
+    StructField("size", StringType, nullable = true),
+    StructField("cluster_size", StringType, nullable = true),
+    StructField("min_num_clusters", LongType, nullable = true),
+    StructField("max_num_clusters", LongType, nullable = true),
+    StructField("auto_stop_mins", LongType, nullable = true),
+    StructField("auto_resume", BooleanType, nullable = true),
+    StructField("creator_name", StringType, nullable = true),
+    StructField("creator_id", LongType, nullable = true),
+    StructField("tags", MapType(StringType, StringType, valueContainsNull = true), nullable = true),
+    StructField("spot_instance_policy", StringType, nullable = true),
+    StructField("enable_photon", BooleanType, nullable = true),
+    StructField("enable_serverless_compute", BooleanType, nullable = true),
+    StructField("warehouse_type", StringType, nullable = true),
+    StructField("num_clusters", LongType, nullable = true),
+    StructField("num_active_sessions", LongType, nullable = true),
+    StructField("state", StringType, nullable = true),
+    StructField("organization_id", StringType, nullable = false),
+    StructField("Pipeline_SnapTS", TimestampType, nullable = true),
+    StructField("Overwatch_RunID", StringType, nullable = true),
```

Comment:
Overwatch_RunID is part of your target key, I don't think it should be nullable. I also think Pipeline_SnapTS shouldn't be nullable either

Reply:
done

Reply:
Overwatch_RunID and PipelineSnapTS will be added after the execution of transformation function, it is being added to the tables using append function. Making it false will fail the module